src/components/Header/index.tsx

```diff
@@ -98,6 +102,7 @@ export default () => {
         )}
       </HeaderPanel>
       <MaintainAlert />
+      {activate ? <NodeAlert /> : null}
```

Comment:
`activate` is a verb but used as an adjective here, I think `isActivated` is more correct, or simply `isOn`